slipcover/__main__.py

```diff
@@ -3,16 +3,17 @@
 from typing import Any, Dict
 from slipcover import slipcover as sc
 from slipcover import bytecode as bc
+from slipcover import branch as br
```

Comment:
consider adding `isort` to your builds to sort the imports.

Reply:
Mm, thanks.